### DISCUSSION TITLE
This is a very bad tool

### DISCUSSION THREAD

**Discussion by korkis** [eslint/eslint#16830] [Ideas]
Most of the time I get my code not working properly because of some space or non-fatal stuff. Or simply comment out the return part for a little test, but I can't check the test result because of a eslint error. 

The eslint I experienced often hindered my coding and ruined my productivity. eslint needs to drastically relax the stringency of the rules by default. This is because many developers set overly strict rules without thinking.

**Comment by nzakas** [comment]
Thanks for the feedback. ESLint is completely customizable, so you can always turn off any rules you don't like. I'd suggest working with your project lead to determine what the best ruleset is for your project so that you create a nice balance between helpful suggestions and productivity.

**Comment by sam3k** [comment]
Hello @korkis, there are some great configures out there such as [AirBnB's config](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) which you can use as a starting point and then do some light tweaking to your liking. It does not take more than an hour of going over it and decide what you like and don't like and turn on and off accordingly.
